packages/salesforcedx-vscode-core/src/commands/forceSourceDeploySourcePath.ts

```diff
@@ -166,3 +100,53 @@ export async function forceSourceDeployMultipleSourcePaths(uris: vscode.Uri[]) {
   );
   await commandlet.run();
 }
+
+// this supported types logic is temporary until we have a way of generating the metadata type from the path
+// once we have the metadata type we can check to see if it is a toolingsupportedtype from that util
+function useBetaRetrieve(explorerPath: vscode.Uri): boolean {
```

Comment:
Everything looks good, are we planning on removing this temporary logic in the next version?

Reply:
yes, we'll be removing this in the future